modules/sno-du-disabling-ntp.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-disabling-ntp_{context}"]
+= Disabling Network Time Protocol (NTP)
+
+After the system is configured for Precision Time Protocol (PTP), you need to
+remove NTP from impacting the system clock.
```

Comment:
you need to remove NTP to prevent it from impacting the system clock.

Reply:
changed.